java/server/src/org/openqa/selenium/grid/node/config/NodeOptions.java

```diff
@@ -70,7 +70,7 @@ private void addSystemDrivers(
       Capabilities caps = info.getCanonicalCapabilities();
       builders.stream()
           .filter(builder -> builder.score(caps) > 0)
-          .peek(builder -> LOG.info(String.format("Adding %s %d times", caps, info.getMaximumSimultaneousSessions())))
+          .peek(builder -> LOG.finest(String.format("Adding %s %d times", caps, info.getMaximumSimultaneousSessions())))
```

Comment:
This is an informational message that allows someone to read the console output and understand how the grid node is configured. Please leave.